src/widget/findonwebfactory.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <QMenu>
+
+class FindOnWeb;
+class Track;
+
+class FindOnWebFactory : public QMenu {
```

Comment:
The factory should not inherit menu. Think on a real factory a t-shirt factory is not a t-shirt itself.

Reply:
Okay, I was having a problem where should I hook the main Find On Web QMenu and how to connect the other services menu with that menu, So this class should only have the logic call the other Services QMenu's or QActions. So I think for the main Find On Web QMenu, I can define it as a QMenu* on WTrackMenu or producer should have it.